boards/hardkernel/odroid_go/odroid_go_appcpu.dts

```diff
@@ -40,27 +40,28 @@
 			read-only;
 		};
 
-		/* Reserve 1024kB for the application in slot 0 */
+		/* Reserve 2048kB for the application in slot 0 */
 		slot0_partition: partition@10000 {
 			label = "image-0";
-			reg = <0x00010000 0x00100000>;
+			reg = <0x00010000 0x00200000>;
 		};
 
-		/* Reserve 1024kB for the application in slot 1 */
-		slot1_partition: partition@110000 {
+		/* Reserve 2048kB for the application in slot 1 */
```

Comment:
you don't need a scratch partition for the MCUboot.

Reply:
Scratch partition removed.

Reply:
@ydamigos, I should be more specific. While the scratch partition is no longer necessary for the MCUboot image manipulations, it is still used in some tests to store data. Size of 64kB should be more than enough.

Reply:
Done